src/bar/mod.rs

```diff
@@ -79,20 +71,20 @@ impl Bar {
 }
 
 unsafe extern "C" fn notify_number_trampoline<P, F: Fn(&P) + 'static>(
-    this: glib_ffi::gpointer,
-    _param_spec: glib_ffi::gpointer,
-    f: glib_ffi::gpointer,
+    this: *mut imp::BarInstance,
+    _param_spec: glib::ffi::gpointer,
+    f: glib::ffi::gpointer,
 ) where
     P: IsA<Bar>,
 {
     let f: &F = &*(f as *const _);
```

Comment:
```suggestion
    let f = &*(f as *const F);
```

What I wrote there before was strange :D

Reply:
ack